pom.xml

```diff
@@ -126,6 +126,11 @@
         <version>1.1.2</version>
         <scope>test</scope>
       </dependency>
+      <dependency>
```

Comment:
what's the root cause here? We shouldn't need this in this repo. The failures are weird as they seem flaky and perhaps dependent on java 8 vs 11. I haven't dug into this yt.

Reply:
I think the problem is that the guava dependencies are using different versions of `error_prone_annotations` (two use 2.3.4 and another one uses 2.5.1), and this causes enforcer failure. This PR specifies the `error_prone_annotations` version in pom.xml so only this version will be used.

```
[WARNING] Rule 2: org.apache.maven.plugins.enforcer.RequireUpperBoundDeps failed with message:
Failed while enforcing RequireUpperBoundDeps. The error(s) are [
Require upper bound dependencies error for com.google.errorprone:error_prone_annotations:2.3.4 paths to dependency are:
+-com.google.http-client:google-http-client:1.38.2-SNAPSHOT
  +-com.google.guava:guava:30.1-android
    +-com.google.errorprone:error_prone_annotations:2.3.4
and
+-com.google.http-client:google-http-client:1.38.2-SNAPSHOT
  +-com.google.guava:guava-testlib:30.1-android
    +-com.google.errorprone:error_prone_annotations:2.3.4
and
+-com.google.http-client:google-http-client:1.38.2-SNAPSHOT
  +-com.google.truth:truth:1.1.2
    +-com.google.errorprone:error_prone_annotations:2.5.1
]
```

Reply:
I sent a cl to update Guava, though it will be some time before that releases.

What I don't understand is why this problem is flaky. Sometimes I see it. Sometimes I don't. It feels like it should be always or never.